BackEnd/src/main/java/com/unosquare/admin_core/back_end/service/EventService.java

```diff
@@ -99,6 +99,15 @@ public void cancelEvent(int eventId){
         }
     }
 
+    public void rejectEvent(int eventId){
+        Optional<Event> retrievedEvent = eventRepository.findById(eventId);
+        if (retrievedEvent.isPresent()) {
+            Event event = retrievedEvent.get();
+            event.setEventStatus(new EventStatus(EventStatuses.REJECTED.getEventStatusId()));
```

Comment:
I don't think we need rejected as a event status, we can make use of denied?

Reply:
No problem.